packages/react-input/src/components/Input/useInputStyles.ts

```diff
@@ -33,49 +33,62 @@ const fieldHeights = {
   medium: '32px',
   large: '40px',
 };
-const borderRadius = (theme: Theme) => theme.borderRadiusMedium;
-const backgroundColors = {
-  /** for outline/filledLighter */
-  filledLighter: (theme: Theme) => theme.colorNeutralBackground1,
-  filledDarker: (theme: Theme) => theme.colorNeutralBackground3,
-  /** for underline */
-  transparent: (theme: Theme) => theme.colorTransparentBackground,
-};
 
 const useRootStyles = makeStyles({
   base: theme => ({
     display: 'flex',
-    alignItems: 'stretch',
+    alignItems: 'center',
     flexWrap: 'no-wrap',
+    gap: horizontalSpacing.xxs,
     fontFamily: theme.fontFamilyBase,
+    borderRadius: theme.borderRadiusMedium, // used for all but underline
     boxSizing: 'border-box',
     '*, *:before, *:after': {
       boxSizing: 'border-box',
     },
   }),
   small: theme => ({
     minHeight: fieldHeights.small,
+    padding: `0 ${horizontalSpacing.sNudge}`,
     ...contentSizes.caption1(theme),
   }),
   medium: theme => ({
-    minHeight: fieldHeights.medium, // minHeight not height in case of tall bookends
+    minHeight: fieldHeights.medium,
+    padding: `0 ${horizontalSpacing.mNudge}`,
     ...contentSizes.body1(theme),
   }),
   large: theme => ({
     minHeight: fieldHeights.large,
+    padding: `0 ${horizontalSpacing.m}`,
     ...contentSizes[400](theme),
   }),
   inline: {
     display: 'inline-flex',
   },
+  outline: theme => ({
+    background: theme.colorNeutralBackground1,
+    border: `1px solid ${theme.colorNeutralStroke1}`,
+    borderBottom: `1px solid ${theme.colorNeutralStrokeAccessible}`,
```

Comment:
You should be able to say `borderBottomColor: theme.colorNeutralStrokeAccessible` so that the border style and weight are controlled by `border` and you don't have to make changes in two places if they ever change.